src/prefect/artifacts.py

```diff
@@ -47,9 +24,11 @@ def create_link(link: str) -> Optional[str]:
     Returns:
         - str: the task run artifact ID
     """
-    if not _running_with_backend():
-        return None
-    return _create_task_run_artifact("link", {"link": link})
+    warnings.warn(
+        "`prefect.artifacts.create_link` has been moved to `prefect.backend.create_link_artifact`. "
+        "Please update your imports. The old function will be removed in 1.0.0."
```

Comment:
should we use the phrasing "This function will be removed" vs. "The old function"?

Reply:
How about
```suggestion
        "Please update your imports. This import path will be removed in 1.0.0."
```